src/OSPSuite.Presentation/Presenters/Importer/UnitsEditorPresenter.cs

```diff
@@ -50,7 +50,9 @@ public void SelectDimension(string dimensionName)
                         _dimensions.FirstOrDefault() ??
                         Constants.Dimension.NO_DIMENSION;
 
-            _selectedUnit = Dimension.DefaultUnit.Name;
+            if (_selectedUnit == null || !Dimension.HasUnit(_selectedUnit))
```

Comment:
I think this would benefit from comments explaining how the unit can not be part of the dimension available units
Also there is a defaultUnitName method I think